src/nomadic/realtime/commands.py

```diff
@@ -233,15 +233,37 @@ def get_region_path(region_bed, workspace):
     return region_bed
 
 
-def get_metadata_path(experiment_name, metadata_csv, workspace):
-    if not metadata_csv:
-        metadata_csv = workspace.get_metadata_csv(experiment_name)
-        if not os.path.isfile(metadata_csv):
-            raise click.BadParameter(
-                message=f"Metadata CSV file not found at {metadata_csv}. Did you create your metadata file in `{workspace.get_metadata_dir()}` and does the name match `{experiment_name}`?",
+def get_metadata_path(experiment_name, metadata_path, workspace):
```

Comment:
Can we type hint this function, as well as the other functions at the bottom of this module?

Also -- not 100% sure this is the best location for this, given we have very similar functions in util/metadata.py

Reply:
I typed it, changed the name and added a docstring, hope this helps.